doc/controls/AutoLayoutControl.md

```diff
@@ -36,6 +36,7 @@ Justify | AutoLayoutJustify | Either Stack, space between children behave like a
 Children |  AutoLayoutChildren | List of FrameworkElement that are the AutoLayout children
 PrimaryAxisAlignment | AutoLayoutAlignment | Indicates where the content be displayed on the axis of the AutoLayout orientation
 IsReverseZIndex | bool | Reverse the Z-index of all the children
+Padding | Thickness | /!\ Auto Layout padding behave like in the figma pluging : The anchors points determine for which sides the padding is taken in consideration. Exemple if the items are align in Right and Top position, only the Right and Top padding value will affect them.   
```

Comment:
```suggestion
Padding | Thickness | /!\ Padding for `AutoLayout` should behave the same as it does within the Figma Plugin: The anchor points determine which sides of the Padding will taken into consideration. For example, items that are aligned to the Right and Top positions will only take the `Tickness.Right` and `Thickness.Top` values of `Padding` into consideration.
```

I cleaned up the grammar here but to be honest I have absolutely no idea what this property is doing. Are we saying that applying a Padding to the AutoLayout control won't behave the same as it would when setting Padding on any other type of control? 

Perhaps we need a better property here, like `PaddingMask` with values of Top, Left Right, Bottom?

Reply:
No, this control mimics figma layout behavior. Given that this is a _temporarily_ control, it is fine to leave it as close as possible to figma, both in properties and in behaviors.
Btw, IIRC it will eventually be replaced by Grid and appropriate properties (in uwp terms).

Reply:
thanks !